.github/workflows/build.yml

```diff
@@ -38,15 +38,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      # Reads versions from root package.json "volta" section
-      - name: Setup Node/NPM with pinned Volta version
-        uses: volta-cli/action@v4.0.0
-
-      # Forces volta to load node and npm upfront, reducing conflict when running in parallel later
-      - name: Preload Node/NPM
-        run: |
-          node -v
-          npm -v
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: package.json
```

Comment:
```suggestion
          node-version-file: ".node-version"
```

Reply:
Since we're not using volta, an additional step is needed to install npm at the proper version as well.

Reply:
Is it really important to pin npm to a specific version? Different users may have a wide range of npm versions 🤔